packages/contracts-bedrock/test/dispute/FaultDisputeGame.t.sol

```diff
@@ -67,7 +68,7 @@ contract FaultDisputeGame_Init is DisputeGameFactory_Init {
         assertEq(gameProxy.absolutePrestate().raw(), absolutePrestate.raw());
         assertEq(gameProxy.maxGameDepth(), 2 ** 3);
         assertEq(gameProxy.splitDepth(), 2 ** 2);
-        assertEq(gameProxy.gameDuration().raw(), 7 days);
+        assertEq(gameProxy.maxClockDuration().raw(), 3.5 days);
```

Comment:
If it's too annoying to regen bindings, in a follow on pr let's add a check here for the clock extension being set to `3 days`.

Reply:
Nah let's do that now, good callout.